web-compile-config.yml

```diff
@@ -0,0 +1,13 @@
+web-compile:
+  sass:
+    files:
+      src/scss/sphinx-book-theme.scss: sphinx_book_theme/static/sphinx-book-theme.[hash].css
+    precision: 5
+    sourcemap: false
+    format: compressed
+  js:
+    files:
+      src/js/sphinx-book-theme.js: sphinx_book_theme/static/sphinx-book-theme.[hash].js
+  jinja:
+    files:
+      src/jinja/theme.conf.j2: sphinx_book_theme/theme.conf
```

Comment:
so this pre-commit hook will generate the new CSS and JS files, + a new `theme.conf` file if the previous files have changed at all?

Reply:
yep, give it a go! thats the best way to see how (if 🤞)  it works

Reply:
and obviously you can have a look at https://github.com/executablebooks/web-compile if you want to understand further